Detector/DetCommon/compact/materials.xml

```diff
@@ -175,6 +175,33 @@
     <fraction n="0.0671091" ref="Br"/>
   </material>
 
+ <!-- Steel for shielding -->
+  <!-- from FLUKA simulations (M.I.Besana) -->
+  <material name="SteelShielding">
+    <D value="7.9" unit="g/cm3" />
+    <fraction n="0.6868" ref="Fe"/>
+    <fraction n="0.17" ref="Cr" />
+    <fraction n="0.032" ref="Mo" />
+    <fraction n="0.11" ref="Ni" />
+    <fraction n="2.E-4" ref="C" />
+    <fraction n="0.001" ref="N" />
+  </material>
+
+  <material name="H_CH2">
+    <D value="15.0345" unit="g/mol" />
+    <fraction n="0.201125" ref="H" />
+    <fraction n="0.798875" ref="C" />
+  </material>
+
+  <material name="lithiatedPolyethylene">
+    <D value="1." unit="g/cm3" />
+    <fraction n="0.12935" ref="H_CH2"/>
+    <fraction n="0.77065" ref="C" />
+    <fraction n="0.001721" ref="Lithium" />
+    <fraction n="0.025029" ref="Lithium" />
```

Comment:
In principle what Ilaria uses is Lithium-6 and Lithium-7, but we only have one implementation, and that's efficient for our purposes. But I could add a comment that makes it clear why we have two numbers.. ?